ocean_lib/models/v4/test/test_dispenser.py

```diff
@@ -0,0 +1,218 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import pytest
+from web3 import exceptions
+
+from ocean_lib.models.v4.dispenser import DispenserV4
+from ocean_lib.models.v4.erc20_token import ERC20Token
+from ocean_lib.models.v4.erc721_factory import ERC721FactoryContract
+from ocean_lib.models.v4.erc721_token import ERC721Token
+from ocean_lib.models.v4.models_structures import DispenserData, ErcCreateData
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from tests.resources.helper_functions import deploy_erc721_erc20, get_address_of_type
+
+
+def test_properties(web3, config):
+    """Tests the events' properties."""
+    dispenser_address = get_address_of_type(config, DispenserV4.CONTRACT_NAME)
+    dispenser = DispenserV4(web3, dispenser_address)
+
+    assert (
+        dispenser.event_TokensDispensed.abi["name"]
+        == DispenserV4.EVENT_TOKENS_DISPENSED
+    )
+    assert (
+        dispenser.event_OwnerWithdrawed.abi["name"]
+        == DispenserV4.EVENT_OWNER_WITHDRAWED
+    )
+    assert (
+        dispenser.event_DispenserAllowedSwapperChanged.abi["name"]
+        == DispenserV4.EVENT_ALLOWED_SWAPPER_CHANGED
+    )
+    assert (
+        dispenser.event_DispenserDeactivated.abi["name"]
+        == DispenserV4.EVENT_DISPENSER_DEACTIVATED
+    )
+    assert (
+        dispenser.event_DispenserActivated.abi["name"]
+        == DispenserV4.EVENT_DISPENSER_ACTIVATED
+    )
+    assert (
+        dispenser.event_DispenserCreated.abi["name"]
+        == DispenserV4.EVENT_DISPENSER_CREATED
+    )
+
+
+# consumer is consumer
+# publisher is publisher
```

Comment:
No need to explain the roles above this function, basically the arguments are self explanatory.